download_list.json

```diff
@@ -1,8 +1,8 @@
 {
 	"downloads": [
 	  {"file":"ePSXe205.zip", "url":"http://www.epsxe.com/files/ePSXe205.zip"},
-	  {"file":"pcsx2-1.6.0-setup.exe", "url":"http://delivery2.filecroco.com/kits_4/pcsx2-1.6.0-setup.exe"},
-	  {"file":"RetroArch.7z", "url": "https://buildbot.libretro.com/stable/1.8.6/windows/x86_64/RetroArch.7z"},
+	  {"file":"pcsx2-1.6.0-binaries.7z", "url":"https://pcsx2.net/download/releases/windows/send/40-windows/125-pcsx2-1-6-0-binaries.html"},
```

Comment:
If you're changing this to be a binary, you'll have to change the way it handles it in the script

Reply:
sorry man, I made a mistake, thanks for the tip
you can revert that if its possible